### ISSUE TITLE
correlation between dependencies and requests doesn't seem to be working correctly in WebApi.Logging

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus.webapi#408]
**Describe the bug**
I'm using Arcus.WebApi.Logging 1.7.0-preview-3 in conjunction with Arcus.Observability 2.6.0

I have a web api where an operation has some dependencies on other resources like an IoT Hub and SQL Server.
When I look in Application Insights on how the request and dependencies are logged, it looks like this:

![image](https://user-images.githubusercontent.com/3605786/204036252-e6df68db-c4f5-425b-8f9d-8c4a62caf617.png)

As you can see, there is no visible parent / child relationship between the request and the dependencies.

When I use the default request logging middleware for application insights, provided by microsoft, the details of my request look like this:

![image](https://user-images.githubusercontent.com/3605786/204036362-1e28765c-d4aa-4d1d-a62a-ff08d7d5dd55.png)

**Comment by stijnmoreels** [comment]
We have to reduce the place where the problem could occur, bc I check with my own application locally with the new preview version, and it worked fine.

**Comment by stijnmoreels** [comment]
It could also help if you could provide the actual child/parent ID's as to pinpoint if there is still a dependency or request using an older version. Thx!

**Comment by fgheysels** [comment]
> How the incoming HTTP request looks like (W3C supported our not)
I'm using Arcus.WebApi 1.7.0-preview-3.  I thought W3C was the default correlation strategy in that version ?

The relevant p roperties for the incoming request look like this:
![image](https://user-images.githubusercontent.com/3605786/204230427-226fc488-116c-4e2f-8da3-5d0040558f75.png)

The relevant properties of one of the outgoing dependencies looks like this:
![image](https://user-images.githubusercontent.com/3605786/204230514-ca020735-6693-4f0c-986e-a02c61d47c52.png)


As I assume W3C is default corelation strategy in 1.7.0-preview-3, in `ConfigureServices` , I have this:

```csharp
builder.Services.AddHttpCorrelation((HttpCorrelationInfoOptions options) => { });
```
This is exactly the same line of code that was present when using an older version of ARcus.WebApi.

and in `ConfigureApp` I just have this:

```
app.UseHttpCorrelation();
```

> How the HTTP correlation is set on the receiving app
There is no receiving app.  I just have one Web API which performs calls to IoT Hub and to SQL Server.

> How the HTTP correlation is added to the Application Insights Serilog sink

> How the HTTP request is tracked on the receiving app
There is no receiving app.

> What version you use of Arcus components (older versions of Observability will not be able to support how we do things with W3C)
I'm using components of Arcus.Observability (2.6.0).

> How the HTTP correlation is registered in the application services
See above.

**Comment by stijnmoreels** [comment]
> I'm using Arcus.WebApi 1.7.0-preview-3. I thought W3C was the default correlation strategy in that version ?

Yes, it is, but if the incoming does not have a `traceparent`, any HTTP dependency could not be added as parent to the child HTTP request.

> There is no receiving app.

I mean how the Web API handles received requests. So, how the HTTP request tracking is configured. You said you were using Serilog before?

It is still strange, as I was using the same kind of setup in another Web API (with multiple API's even) and got it working rather fastly.

**Comment by stijnmoreels** [comment]
I'll have to recheck some things to see if I can reproduce this.

**Comment by fgheysels** [comment]
I think it might be good to setup a short call for this.

**Comment by fgheysels** [comment]
I've tested this again (with 1.7.0 preview 4), but I see no difference.  The issue still exists

![image](https://user-images.githubusercontent.com/3605786/204807546-0f4ed5e2-8fa6-435d-892f-e0b4921e06d7.png)

**Comment by fgheysels** [comment]
I think this can be closed ?
